librz/diff/distance.c

```diff
@@ -0,0 +1,105 @@
+// SPDX-FileCopyrightText: 2017 Fangrui Song <i@maskray.me>
+// SPDX-FileCopyrightText: 2016 NikolaiHampton <nikolaih@3583bytesready.net>
+// SPDX-License-Identifier: LGPL-3.0-only
+#include <rz_diff.h>
+#include <rz_util/rz_assert.h>
+
+// Eugene W. Myers' O(ND) diff algorithm
```

Comment:
Maybe convert this comment into Doxygen description too, since it's here anyway...